proto/list_hosts.proto

```diff
@@ -34,10 +34,26 @@ message HostList {
         int32 vlan = 4;
 
         // ACLs
-        repeated string acls = 5;
+        repeated ACLData acls = 5;
 
         // link to list_hosts API to list possible destinations
         // or to host_path API to list path
         string url = 6;
     }
-}
\ No newline at end of file
+
+    message ACLData {
+        // acl name
+        string name = 1;
+
+        // rule states indexed by rule cookies
+        repeated RuleData rules = 2;
+
+        message RuleData {
```

Comment:
I don't understand what this is -- why are there no counts here?  How/why is this different than ACLState?

Reply:
Using RuleState now, to be consistent with switch_state API.